content/about/contribute/diagrams/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Diagram Creation Guidelines
+description: Provides assets and instructions to create diagrams for the Istio documentation.
+weight: 40
+keywords: [contribute,diagram,documentation,guide]
+---
+
+Welcome to the Istio diagram guide!
+
+The guide is available as an [SVG file](./diagram-guidelines.svg) or as a
+[Google Draw file](https://docs.google.com/drawings/d/1f3NyutAQIDOA8ojGNyMA5JAJllDShZGQAFfdD01XdSc/edit)
+to allow you to reuse the shapes and styles with ease. Use these guidelines to
+create SVG diagrams for the Istio website using any vector graphics tool like
+Google Draw, InkScape, or Illustrator. Please ensure that the text in your
+diagrams remains editable.
+
+Our goal is to drive consistency across all diagrams in our website to ensure
+diagrams are clear, technically accurate, and accessible.
+
+Keeping the text editable allows the community to improve and change the
+diagrams as needed.
+
+Follow these steps to begin using the guideline:
+
+1. [Download the guidelines as an SVG file.](./diagram-guidelines.svg)
+1. Select the shapes you want to use.
+1. Copy them into a new SVG file.
+1. Make as many duplicates as you need for your diagram.
+1. Connect the shapes with the appropriate style of line.
```

Comment:
I think the first four bullets are bit too hand-holdy for folks. Can we just summarize in one sentence:

Use the diagram for guidance, cut and paste from it as needed, and then follow these steps: